src/main/java/ch/aaap/harvestclient/domain/reference/dto/ProjectReferenceDto.java

```diff
@@ -10,4 +10,6 @@
 @Value.Immutable
 public interface ProjectReferenceDto extends BaseReferenceDto, Reference<Project> {
 
+   String getCode();
```

Comment:
```suggestion
@Nullable
   String getCode();
```
This code is optional